trash.go

```diff
@@ -435,14 +443,18 @@ func cloneGitRepo(trashDir, repoDir string, i conf.Import, insecure bool) error
 	return nil
 }
 
-func fetch(i conf.Import) error {
+func fetch(i conf.Import, retry int) (err error) {
 	remote := remoteName(i.Repo)
-	logrus.Infof("Fetching latest commits from '%s' for '%s'", remote, i.Package)
-	if bytes, err := exec.Command("git", "fetch", "-f", "-t", remote).CombinedOutput(); err != nil {
-		logrus.Errorf("`git fetch -f -t %s` failed:\n%s", remote, bytes)
-		return err
-	}
-	return nil
+	rtr := retrier.New(retrier.ConstantBackoff(retry, time.Second), retrier.DefaultClassifier{})
+	err = rtr.Run(func() error {
```

Comment:
Can you remove the need for an external library here? I think just a simple for loop would work.